context/AuthContext.js

```diff
@@ -0,0 +1,71 @@
+import React,{createContext,useEffect,useState} from 'react';
+import axios from 'axios';
+import { BASE_URL } from '@env';
+import {LoginEndpoint,SignupEndpoint,UserFile,TokenFile} from '../APIs.js'
+// import EncryptedStorage from 'react-native-encrypted-storage';
+import {Storage} from 'expo-storage'
+import * as FileSystem from 'expo-file-system';
+
+export const AuthContext = createContext();
+
+export const AuthProvider = ({children}) =>{
+    const [isLoading,setIsLoading]=useState(false)
+    const [userToken,setUserToken]=useState(null)
+    const [user,setUser]=useState(null)
```

Comment:
I think it would be better to initialize with empty string and empty object instead of `null` to avoid errors from accessing empty objects in the future, let me know what you think